src/main/scala/is/hail/stats/CallStatsCombiner.scala

```diff
@@ -9,27 +9,33 @@ object CallStats {
   def schema = TStruct(
     "AC" -> TArray(TInt32()),
     "AF" -> TArray(TFloat64()),
-    "AN" -> TInt32())
+    "AN" -> TInt32(),
+    "homozygote_count" -> TArray(TInt32()))
 }
 
-case class CallStats(alleleCount: IndexedSeq[Int], alleleFrequency: Option[IndexedSeq[Double]], alleleNumber: Int) {
+case class CallStats(alleleCount: IndexedSeq[Int], alleleFrequency: Option[IndexedSeq[Double]], alleleNumber: Int,
+  homCount: IndexedSeq[Int]) {
   require(alleleFrequency.forall(f => D_==(f.sum, 1d)), s"AF did not sum to 1: $this")
-
-  def asAnnotation: Annotation = Annotation(alleleCount, alleleFrequency.orNull, alleleNumber)
+  require(homCount.sum >= 0 && homCount.sum <= alleleNumber.toDouble / 2)
+  def asAnnotation: Annotation = Annotation(alleleCount, alleleFrequency.orNull, alleleNumber, homCount)
 }
 
 class CallStatsCombiner(val alleles: IndexedSeq[String]) extends Serializable {
```

Comment:
As far as I can tell, this only uses alleles.length.  It should only take what it needs, which should simplify some upstream code.

Reply:
Heh, I didn't notice this while I was reading the PR.